crosschain/src/main/java/org/hyperledger/besu/crosschain/core/CrosschainProcessor.java

```diff
@@ -54,8 +58,13 @@
 
   private LinkedNodeManager linkedNodeManager;
 
-  public CrosschainProcessor(final LinkedNodeManager linkedNodeManager) {
+  @SuppressWarnings("UnusedVariable")
```

Comment:
Is this reallly necessary?

Reply:
Definitely not. This is a spill over from the time when I was debugging. Thanks for spotting it.